tests/foreman/ui_airgun/test_partitiontable.py

```diff
@@ -0,0 +1,211 @@
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'snippet': False,
+            'os_family_selection': {'os_family': 'Debian'},
+            'template': template,
+        }, )
+        pt = session.partitiontable.read(name)
+        assert pt['os_family_selection']['os_family'] == 'Debian'
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_with_snippet(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'snippet': True,
+            'template': template
+        }, )
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_with_audit_comment(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    audit_comment = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'audit_comment': audit_comment,
+            'template': template
+        }, )
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_loc_org(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    locations = entities.Location().create()
+    organizations = entities.Organization().create()
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'locations.assigned': [locations.name],
+            'organizations.assigned': [organizations.name],
+            'template': template
+        })
+        pt = session.partitiontable.read(name)
+        assert locations.name in pt['locations']['assigned']
+        assert organizations.name in pt['organizations']['assigned']
+
+
+def test_positive_delete_with_lock_and_unlock(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    audit_comment = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'default': True,
+            'snippet': True,
+            'audit_comment': audit_comment,
+            'template': template
+        }, )
+        assert session.partitiontable.search(name)[0]['Name'] == name
+        session.partitiontable.lock(name)
+        try:
+            session.partitiontable.delete(name)
+        except ValueError:
+            assert session.partitiontable.search(name)[0]['Name'] == name
+        session.partitiontable.unlock(name)
+        session.partitiontable.delete(name)
+        assert not session.partitiontable.search(name)
+
+
+def test_positive_create_default_for_organization(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    default_org = entities.Organization().search(
+        query={'search': 'name="{0}"'.format(DEFAULT_ORG)})[0]
```

Comment:
this api command is redundant - you only use a name of default organization in your test, and to get the name you don't have to a search - just use `DEFAULT_ORG` constant directly :)